server/public/Makefile

```diff
@@ -0,0 +1,14 @@
+test-public: gotestsum
```

Comment:
Do we need a discrete `test-public`? This should all get tested from the root `Makefile` already, and when one needs to run a subset of tests, it's already necessary to `go test` (or `gotestsum` ❤️ ) manually.

Reply:
To me, developing in the `public/plugin` package (and its sub-packages) is a separate workflow that deserves its own make target 2/5. It's only a convenience wrapper around `go test` (or `gotestsum`), which makes it easier for devs to run their tests.

Reply:
As pointed out in https://community.mattermost.com/core/pl/orm4ewwmwb8wbpquou6my4747a, `make test-server` does not run the tests in `public`.